product_state/__init__.py

```diff
@@ -1,3 +1,17 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from . import models
+
+
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
```

Comment:
I would prefer this to be in it's own file, but not a blocking issue.

Reply:
Done